### DISCUSSION TITLE
Resize image by factor of cos(x)

### DISCUSSION THREAD

**Discussion by brighamcampbell** [ImageMagick/ImageMagick#6378] [Help]
### ImageMagick version

7.1.1-9

### Operating system, version and so on

Arch Linux

### Description

Hello!

I want to resize the y axis of my image by a factor of cos(x) in my script. Is it possible to do this without calculating the value beforehand? Something like `-fx` for just one scalar value?

**Comment by fmw42** [comment]
In IM 7, -resize supports the use of "%wx%[fx:h*cos($value)]" as the argument, where "value" is some constant in radians.  This keeps the input w (width) the same and scales the input h (height) by the cos.
